common/djangoapps/student/tests/test_reset_password.py

```diff
@@ -287,6 +288,23 @@ def test_password_reset_fail(self):
         self.assertEqual(resp.status_code, 200)
         self.assertFalse(User.objects.get(pk=self.user.pk).is_active)
 
+    @override_settings(PASSWORD_MIN_LENGTH=2)
+    def test_password_reset_with_min_length(self):
```

Comment:
I feel like it's worth the effort to test both the min length and max length part of this functionality.
Just override the setting with `PASSWORD_MAX_LENGTH=10`, then use `ddt` to create a set of passwords to reset on and inspect the error messages.